test/functional/lua/fs_spec.lua

```diff
@@ -454,4 +454,38 @@ describe('vim.fs', function()
       end)
     end)
   end)
+
+  describe('abspath', function()
+    local cwd = is_os('win') and vim.uv.cwd():gsub('\\', '/') or vim.uv.cwd()
+    local home = is_os('win') and vim.uv.os_homedir():gsub('\\', '/') or vim.uv.os_homedir()
```

Comment:
`fix_slashes` https://github.com/neovim/neovim/blob/487c48ec8689b865bad04fdb87b61f5ada25da97/test/testutil.lua#L50

Reply:
Fix in https://github.com/neovim/neovim/pull/31790

Reply:
I have no idea why that function accepts a table. We definitely shouldn't port that aspect over, nor do I think we need to expose such a function.

Reply:
It's just a util function for tests. Tests should use it instead of doing it manually.

Reply:
But does that mean we have to expose this as stable API? It's just a basic gsub. Doesn't seem like the tradeoffs are being properly considered.

Reply:
`fix_slashes` isn't being exposed (AFAIK).

are you commenting on `abspath` ?

Reply:
Yeah, what is bro on about

Reply:
Not exposing `fix_slashes` in my PR.

Reply:
I'm commenting specifically about `fix_slashes`, as mentioned in the first comment here.

Reply:
ngl, I'm kinda confused as well. We're not exposing or porting `fix_slashes`, it is only used in tests.

Reply:
Yeah sorry, I'm going to finger point GitHub mobile for this. Backing out of the notification and viewing the full code, gave me the context I was missing, mainly the filename and that this was a test. Sorry for the misunderstanding.